src/components/pages/import.js

```diff
@@ -0,0 +1,158 @@
+import by from 'thenby';
+import Page from '../page.js';
+import PropTypes from 'prop-types';
+import urls from '../../utils/urls.js';
+import React, { useState } from 'react';
+import Logger from '../../common/logger.js';
+import { useTranslation } from 'react-i18next';
+import { Tooltip, Button, Table, Select, Spin } from 'antd';
+import { inject, useService } from '../container-context.js';
+import { DOCUMENT_IMPORT_TYPE } from '../../common/constants.js';
+import ImportApiClient from '../../services/import-api-client.js';
+import { useDateFormat, useLanguage } from '../language-context.js';
+import LanguageNameProvider from '../../data/language-name-provider.js';
+import CountryFlagAndName from '../localization/country-flag-and-name.js';
+import { CloudDownloadOutlined, CloudSyncOutlined } from '@ant-design/icons';
+
+const logger = new Logger(import.meta.url);
+
+function Import({ initialState, importApiClient }) {
+  const { t } = useTranslation('import');
+
+  const { language } = useLanguage();
+  const { formatDate } = useDateFormat();
+  const languageNameProvider = useService(LanguageNameProvider);
+
+  const sourceMenuItems = initialState.importSources;
+  const [selectedSource, setSelectedSource] = useState(null);
+  const [selectedDocumentKeys, setSelectedDocumentKeys] = useState([]);
+  const [importableDocuments, setImportableDocuments] = useState([]);
+
+  const handleImportClick = () => {
+    const tasks = selectedDocumentKeys
+      .map(key => importableDocuments.find(doc => doc.key === key))
+      .map(doc => ({
+        key: doc.key,
+        importedRevision: doc.importedRevision,
+        importableRevision: doc.importableRevision
+      }));
+    logger.info(`Dummy import sets tasks: ${JSON.stringify(tasks)}`);
+  };
+
+  const handleSourceMenuChange = async value => {
+    const newSelectedSource = sourceMenuItems.find(source => source.name === value);
+    setSelectedSource(newSelectedSource);
+    setImportableDocuments([]);
+    setSelectedDocumentKeys([]);
+
+    const { documents } = await importApiClient.getImports(newSelectedSource);
+    setImportableDocuments(documents);
+  };
+
+  const renderImportType = doc => {
+    let icon = doc.importType;
+    if (doc.importType === DOCUMENT_IMPORT_TYPE.add) {
+      icon = <CloudDownloadOutlined />;
+    }
+    if (doc.importType === DOCUMENT_IMPORT_TYPE.update) {
+      icon = <CloudSyncOutlined />;
+    }
+    return <Tooltip title={t(doc.importType)} className="ImportPage-importType">{icon}</Tooltip>;
+  };
+
+  const renderTitle = doc => {
+    if (!doc.slug) {
```

Comment:
I basically had the same question.
Since we'll want to move towards having slugs just for SEO and unify doc routes to use IDs only, then I'll change all of them here to use the ID consistently. Thanks for the reminder, had forgotten.

Reply:
@ahelmberger What do you think, shall we do this ☝️ ?

Reply:
Maybe not to the edit page, rather to the doc page itself, but to be honest, "nu slug no luck", we can also just not render a link at all.

Reply:
I think we promised a link in the planning.

Reply:
But documents without slugs cannot be linked at the moment. I wouldn't consider it a holy promise, more like a "if-possible-to-achieve-for-humans" kinda promise ;-)

Reply:
Ok, cool then.